webdriverio/index.d.ts

```diff
@@ -1082,6 +1109,7 @@ declare namespace WebdriverIO {
 
 declare var browser: WebdriverIO.Client<void>;
 
+// tslint:disable-next-line:no-single-declare-module
```

Comment:
Since this should be fixed eventually, please add `"rules": { "no-single-declare-module": false }` to the `tslint.json` as a reminder to deal with it later.